src/tiler/server.js

```diff
@@ -24,7 +24,7 @@ var config = {
         }
     },
     renderCache: {
-        ttl: 60000, // seconds
```

Comment:
This seems like a pretty big change to make without adding an explanation of why it was necessary in the commit message, might have some implications worth discussing.